packages/components/src/draggable/index.native.js

```diff
@@ -27,7 +31,7 @@ import { Platform } from '@wordpress/element';
 export default function Draggable( {
 	children,
 	maxDistance = 1000,
-	minDuration = 500,
+	minDuration = 450,
```

Comment:
This change is only necessary for iOS. In the future, we might consider adding a similar workaround as the one introduced for Android ([reference](https://github.com/WordPress/gutenberg/blob/1d9c0fc7bc35a7c8728cabf5eda23213d0bb1eff/packages/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java#L148-L157)), and enable/disabe the long-press gesture from the native side.